dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceServiceTest.java

```diff
@@ -342,4 +342,25 @@ void testTrackedEntityInstanceGridWithNoDisplayAttributes()
         assertThrows( IllegalArgumentException.class,
             () -> entityInstanceService.getTrackedEntityInstancesGrid( params ) );
     }
+
+    @Test
+    void shouldCountOneEntityWhenOnePresent()
+    {
+        entityInstanceA1.setTrackedEntityType( trackedEntityTypeA );
+        entityInstanceService.addTrackedEntityInstance( entityInstanceA1 );
+
+        TrackedEntityInstanceQueryParams params = new TrackedEntityInstanceQueryParams();
+        int trackedEntitiesCounter = entityInstanceService.getTrackedEntityInstanceCount( params, true, true );
```

Comment:
`trackedEntitiesCounter` is a pretty long name while the variable has a short life from declaration to usage. `count` would suffice and make it easier to read IMHO.